src/julia_gc.c

```diff
@@ -575,7 +591,7 @@ static void GapRootScanner(int full)
 {
     // mark our Julia module (this contains references to our custom data
     // types, which thus also will not be collected prematurely)
-    JMark(Module);
+    jl_gc_mark_queue_obj(JuliaTLS, (jl_value_t *) Module);
```

Comment:
Why this change?

Reply:
This allowed me to put validation logic in `JMark()` to verify that bad pointers indeed don't get traversed. I should actually put that in.